conf/layer.conf

```diff
@@ -8,6 +8,6 @@ BBFILE_COLLECTIONS += "synopsys-layer"
 BBFILE_PATTERN_synopsys-layer = "^${LAYERDIR}/"
 BBFILE_PRIORITY_synopsys-layer = "6"
 
-LAYERSERIES_COMPAT_synopsys-layer = " gatesgarth hardknott"
+LAYERSERIES_COMPAT_synopsys-layer = " gatesgarth hardknott honister"
```

Comment:
But I guess that change immediately breaks compatibility with all older versions like `gatesgarth` & `hardknott`, so maybe not extend but replace all with `honister`?

Reply:
Updated the commit